src/usr/local/www/widgets/widgets/thermal_sensors.widget.php

```diff
@@ -34,7 +34,7 @@
 		$_gb = exec("/sbin/sysctl -q dev.cpu | grep temperature | sort",
 		    $dfout);
 	} else {
-		$_gb = exec("/sbin/sysctl -aq | grep temperature", $dfout);
+		$_gb = exec("/sbin/sysctl -aq | grep temperature:", $dfout);
```

Comment:
That's still too general for my taste. Try this:

```php
$_gb = exec("/sbin/sysctl -aq | /usr/bin/egrep '(hw.acpi.thermal|dev.cpu).*\.temperature:'", $dfout);
```

Or, perhaps even better:

```php
$_gb = exec("/sbin/sysctl -aq hw.acpi.thermal dev.cpu | /usr/bin/grep 'temperature:'", $dfout);
```

Reply:
That works for me. The 'a' is unused there though.